conmon-rs/server/Cargo.toml

```
@@ -31,6 +31,7 @@ prctl = "1.0.0"
 regex = "1.9.6"
 sendfd = { version = "0.4.3", features = ["tokio"] }
 serde = { version = "1.0.188", features = ["derive"] }
+serde_json = "1.0"
```

Please use the full version here
```suggestion
serde_json = "1.0.107"
```